### PR TITLE
Tdd

### PR DESCRIPTION
**Pull request by 1akshat**
Solution for test-driven development exercise completed.

### CODE REVIEW

File: js-exercises/least-common-multiple/leastCommonMultiple.js [hunk 1]
```diff
@@ -4,6 +4,8 @@ function leastCommonMultiple(number1, number2) {
 	if(typeof(number1) !== "number" || typeof(number2) !== "number") throw new Error(`Expected both numbers, Got ${typeof number1}, ${typeof number2}`)
```
Comment (Gauravkr2506) [inline js-exercises/least-common-multiple/leastCommonMultiple.js:4]: Good to see that you have validated your incoming data

File: js-exercises/least-common-multiple/leastCommonMultiple.js [hunk 2]
```diff
@@ -4,6 +4,8 @@ function leastCommonMultiple(number1, number2) {
...
 	let large = Math.max(number1, number2);
   let small = Math.min(number1, number2);
+  if(small == 0) return large
+
   let lcm = large;
   while(lcm % small !== 0) {
```
Comment (Gauravkr2506) [inline js-exercises/least-common-multiple/leastCommonMultiple.js:10]: The overall program is good, but I have a suggestion, that this program is working for only two parameters, you can also make it work for extra parameters if you want.

**Review by Gauravkr2506 — COMMENTED** [review]
good job

File: js-exercises/least-common-multiple/leastCommonMultiple.js (hunk 2 above)
Comment (1akshat) [reply js-exercises/least-common-multiple/leastCommonMultiple.js:10]: yes. thanks for the feedback Gaurav.

**Pull request merged**